pkg/kubelet/dockertools/docker.go

```diff
@@ -126,7 +126,12 @@ func (p dockerPuller) Pull(image string) error {
 		Tag:        tag,
 	}
 
-	creds, haveCredentials := p.keyring.Lookup(repoToPull)
+	keyring, err := credentialprovider.MakeDockerKeyring(secrets, p.keyring)
```

Comment:
Looks like this has been addressed to me, looking at the changes to rkt.go in this pull.